File: Hour 6/Q1/salsa_class.c
#include<stdio.h>
#include<string.h>

void main(){
    char queue_A[20];
    gets(queue_A);
    char queue_B[20];
    gets(queue_B);
    int len = strlen(queue_A), i, count = 0;
    for(i = 0; i < len; i++){
        if(queue_A[i] != queue_B[i])
            count++;
    }
    printf("%d",count*100/len);
}

File: Hour 6/Q1/salsa_class.java
import java.util.Scanner;

public class salsa_class{
    public static void main(String[] args) {
        Scanner sc = new Scanner(System.in);
        try{
            String queue_A = sc.nextLine();
            String queue_B = sc.nextLine();
            int len = queue_A.length(), i, count = 0;
            for(i = 0; i < len; i++){
                if(queue_A.charAt(i) != queue_B.charAt(i))
                    count++;
            }
            System.out.print(count*100/len);
        }
        finally{
            sc.close();
        }
    }
}